support-frontend/assets/components/thankYou/thankYouModuleData.tsx

```diff
@@ -47,17 +52,32 @@ interface ThankYouModuleData {
 	trackComponentLoadId?: string;
 }
 
+const defaultSupportReminder = {
+	selectedChoiceIndex: 0,
+	hasBeenCompleted: false,
+	errorMessage: '',
+};
+const defaultFeedbackSurveyHasBeenCompleted = false;
+
 export const getThankYouModuleData = (
 	countryId: IsoCountry,
 	countryGroupId: CountryGroupId,
 	csrf: CsrfState,
-	email: string,
 	isOneOff: boolean,
 	amountIsAboveThreshold: boolean,
+	email?: string,
 	campaignCode?: string,
+	supportReminder?: ThankYouSupportReminderState,
+	feedbackSurveyHasBeenCompleted?: boolean,
 ): Record<ThankYouModuleType, ThankYouModuleData> => {
-	const { feedbackSurveyHasBeenCompleted, supportReminder } =
-		useContributionsSelector((state) => state.page.checkoutForm.thankYou);
+	const initialFeedbackSurveyHasBeenCompleted =
+		feedbackSurveyHasBeenCompleted ?? defaultFeedbackSurveyHasBeenCompleted;
+	const [feedbackSurveyCompleted, setFeedbackSurveyCompleted] =
+		useState<boolean>(initialFeedbackSurveyHasBeenCompleted);
+	const [supportRemind, setSupportRemind] =
```

Comment:
Non-blocking but you could use the same naming convention here as the current convention makes it a little hazy to work out what's going on.